src/tests/ftest/container/autotest.yaml

```diff
@@ -0,0 +1,17 @@
+hosts:
+  test_servers:
+    - server-A
+  test_clients:
+    - server-A
```

Comment:
Shouldn't this be "client-B" instead? Like this:

hosts:
    test_servers:
        - server-A
    test_clients:
        - client-B

Reply:
Good catch, will fix